tests/integ/test_udf.py

```diff
@@ -543,126 +565,125 @@ def test_register_from_file_with_skip_upload(session, resources_path, caplog):
         Utils.drop_stage(session, stage_name)
 
 
+@pytest.mark.localtest
 def test_add_import_local_file(session, resources_path):
     test_files = TestFiles(resources_path)
-    # This is a hack in the test such that we can just use `from test_udf import mod5`,
-    # instead of `from test.resources.test_udf.test_udf import mod5`. Then we can test
-    # `import_as` argument.
-    with patch.object(
-        sys, "path", [*sys.path, resources_path, test_files.test_udf_directory]
-    ):
```

Comment:
I don't have context on why this hack was added and why is it removed now. Can you give more context here

Reply:
As long as the test passes, then it should be ok. Previously I remember if I don't add this patch, the test will fail